wandb/apis/public/artifacts.py

```diff
@@ -508,21 +508,216 @@ def description(self):
         """A description of the artifact collection."""
         return self._description
 
+    @description.setter
+    def description(self, description: Optional[str]) -> None:
+        self._description = description
+
     @property
     def tags(self):
         """The tags associated with the artifact collection."""
         return self._tags
 
+    @tags.setter
+    def tags(self, tags: List[str]) -> None:
+        if any(char in tag for tag in tags for char in ["/", ":"]):
+            raise ValueError(
+                "Tags must not contain any of the following characters: /, :"
+            )
```

Comment:
https://github.com/wandb/core/blob/3a7897907741334bf8848722fdf2bf30f3827527/services/gorilla/schema.graphql#L6109-L6114

If we're displaying a warning here, could you make sure the constraints match the ones defined in the graphql schema for tags? 

You can also refer to the UI code for the list of special characters that aren't allowed.

Reply:
just changed this to throw an error if it doesn't match the constraints in the graphql schema, that should also cover the special characters